GitDiffMargin.Commands/CommandHandlerTextViewCreationListener.cs

```diff
@@ -30,8 +31,8 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
 
             // The new command handling approach does not require that the command filter be enabled. The command
             // implementations interact directly with the handler via its IOleCommandTarget interface.
-            GitDiffMarginCommandHandler filter = new GitDiffMarginCommandHandler(textViewAdapter, _editorAdaptersFactoryService, textView);
+            var filter = new GitDiffMarginCommandHandler(textViewAdapter, _editorAdaptersFactoryService, textView);
             textView.Properties.AddProperty(typeof(GitDiffMarginCommandHandler), filter);
         }
     }
-}
+}
```

Comment:
:memo: I don't like how the missing newline at the end is presented by tools to look like an error.